add go-consistent linter into CI

Run https://github.com/Quasilyte/go-consistent during CI checks.
Note that there is at least 1 warning caught by it right now:
```
checkers/unnamedResult_checker.go:47:7: non-zero length test: use `len(s) != 0`
```
Should be fixed along the way.

--- a/Makefile
+++ b/Makefile
@@ -25,6 +25,8 @@ ci-tests:
 ci-linter:
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b ${GOPATH}/bin v1.12.3
 	golangci-lint run -v
+	go get -u github.com/Quasilyte/go-consistent
+	go-consistent ./...
 
 cover:
 	go get -u github.com/mattn/goveralls

--- a/checkers/unnamedResult_checker.go
+++ b/checkers/unnamedResult_checker.go
@@ -44,7 +44,7 @@ func (c *unnamedResultChecker) VisitFuncDecl(decl *ast.FuncDecl) {
 	switch {
 	case results == nil:
 		return // Function has no results
-	case len(results.List) > 0 && results.List[0].Names != nil:
+	case len(results.List) != 0 && results.List[0].Names != nil:
 		return // Skip named results
 	}
 

